### ISSUE TITLE
Better meta integration

### ISSUE THREAD

**Issue by jvstrange** [b0tchsec/b0tchsec.github.io#10]
I shared the homepage with someone on facebook today and it grabbed one of the pictures from the lossless challenge and the blockquote description from lossless to show in the preview thing that facebook does. We should integrate meta tags into the theme that grab some front-matter and fill it out. Better SEO, better share-ability, etc.

**Comment by ajacques** [comment]
We can integrate the OpenGraph meta tags from: http://ogp.me/

**Comment by jvstrange** [comment]
Yeah I figured we'd want to integrate OG meta tags as well as the
original meta author, description, keywords tags.
